### DISCUSSION TITLE
Capture to a stream instead of file

### DISCUSSION THREAD

**Discussion by mnordstr** [NiiightmareXD/windows-capture#33] [Q&A] [answered]
Hi,

The lib looks nice. Any way to capture the video to a stream instead of writing it directly to file? Would be useful to e.g. send it over the network.

**Comment by NiiightmareXD** [comment]
By default, the API is a stream that was redirected to a file. Do you think it would be better to redirect it to a callback function?

**Reply by mnordstr** [reply]
That could be an idea, or ability to provide the function an IRandomAccessStream instead of a file name. Then the caller could provide the function with a file stream or another stream as needed.

**Reply by NiiightmareXD** [reply, marked as answer]
I added `VideoEncoder::new_from_stream`.

**Reply by mnordstr** [reply]
> I added `VideoEncoder::new_from_stream`.

Excellent! I will give it a try as soon as possible. Thanks!
